docs/source/rf1d_tutorial.rst

```diff
@@ -0,0 +1,216 @@
+
+  --action
+
+This is where you specify what kind of visualization you want. The current options are:
+
+1. *barplot* - This will create a barplot of the rFon1D values, grouped by site and alphabet input. This is called automatically when you run *orthogonal-polynomial*.
+2. *density* - This will create a density plot of the rFon1D values.
+3. *summary* - Prints out the number of sites and dimensions, the alphabet input, the molecule, and calls *sort* (another *rf1d-viz* action that is explained in further detail below). This is called in *orthogonal-polynomial* automatically, and will not be saved.
+4. *heatmap* - This will create a heatmap of the rFon1D values, grouped by site and alphabet input.
+5. *boxplot* - This will create a boxplot of the rFon1D values, grouped by .
+6. *sort* - This will print out the top 10 rFon1D values by magnitude, including the rFon1D value, the site, and the group it belongs to. This will not be saved to the *out_dir*.
+7. *ALL* - This will produce a barplot, histogram, heatmap, and boxplot simultaneously.
+
+    **Note:** For now, you will need to close the first graph once it displays on your computer for the rest of the graphs to run.
+
+.. _input:
+3. Running *rf1d-viz*
+-----------------------------------------------------------
+
+Similarly to *orthogonal-polyomial*, you will run *rf1d-viz* in your CLI, first starting with the keyword *ortho_seq*, but now followed by *rf1d-viz*, instead of *orthogonal-polynomial*. The general format is
+
+.. code-block:: shell-session
+
+  ortho_seq rf1d-viz filename --alphbt_input --molecule --phenotype --out_dir --action
+
+where *filename* represents the *--filename* flag.
+
+.. _example:
+Guided example with the Sidhu dataset
+-----------------------------------------------------------
+
+The example uses the *Sidhu* dataset, which is the same as was used for the *orthogonal-polynomial* tutorial. Recall that the input for *orthogonal-polynomial* was:
+
+.. code-block:: shell-session
+
+  ortho_seq orthogonal-polynomial ortho_seq_code/Sidhu/Sidhu.xlsx --molecule protein --poly_order first --out_dir docs/source/tutorial_outputs --alphbt_input SYG,R --min_pct 40 --pheno_name IC50
+
+The regression file that will be used for *rf1d-viz* will thus be called
+
+.. code-block:: shell-session
+
+  Sidhu_regressions.npz
+
+Using the CLI output, we obtain
+
+.. code-block:: shell-session
+
+  rf1d form of alphabet input:
+  SYG,R,z,n
+
+which reveals that the *rf1d* form of the alphabet input is **SYG,R,z,n**.
+
+With these in mind, the CLI input for *rf1d-viz* for a **barplot** will be
+
+.. code-block:: shell-session
+
+  ortho_seq rf1d-viz docs/source/tutorial_outputs/Sidhu_regressions.npz --alphbt_input SYG,R,z,n --molecule protein --phenotype IC50 --out_dir docs/source/tutorial_outputs --action barplot
+
+This line of code will reproduce the graph that is automatically run, and looks like
+
+.. image:: tutorial_outputs/rFon1D_Regressions_of_IC50_values.png
```

Comment:
Can we change all the plots in this tutorial to say `ELISA` values instead of IC50 values?

Reply:
Please update the y-axes on all the plots. Maybe you could add this part in the tutorial! You could say you realized you needed to rename the phenotype, but you can actually do that now with the --phenotype flag and no need to recompute everything.